garden-backend-service/src/api/routes/entrypoint.py

```diff
@@ -22,6 +22,7 @@ async def add_entrypoint(
     db.add(new_entrypoint)
     try:
         await db.commit()
+        await db.refresh(new_entrypoint)
```

Comment:
this seems reasonable - was this necessary to fix a bug with the tests or just better practice?

Reply:
I thought some of the entrypoint tests were failing due to the relationships not being updated when adding a new one to the database. Turns out that wasn't the issue and the call to refresh is unnecessary. The tests pass without it.